### ISSUE TITLE
[A11Y] Pagination markup should be adjusted to improve accessibility

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#6206]
## Description of the bug

Our pager markup has been reviewed, and could use some improvements.  

Suggested markup is as below, but significant changes are as follows. Some of these items can be achieved in Backdrop 1.x, but others will need to wait until 2.x, as they would be braking changes for existing sites.

* 2.x: add a surrounding `nav` element with `aria-label="Pagination"`
* 1.x Add `aria-label` and `aria-current` attributes on individual page links
* 2.x use `button` elements for First / Last / Next / Previous
* 1.x Add total count of pages for mobile `<p>Page 1 of 10</p>`
* 1.x: Add `display:none` to specific elements to create desktop vs mobile 

## Mobile recommendation:
```
<nav aria-label="Pagination">
  <button>Previous</button><!-- can be disabled (not removed) if this is the first -->
  <p>Page 1 of 10</p>
  <button>Next</button><!-- can be disabled (not removed) if this is the last -->
</nav>
```

## desktop recommendation
```
<nav aria-label="Pagination">
  <button>Previous</button><!-- can be disabled (not removed) if this is the first -->
  <button>First</button><!-- can be disabled (not removed) if this is the first -->
  <ul>
    <li><a href="/1" aria-label="Page 1" aria-current="page">1</a></li>
    <li><a href="/2" aria-label="page 2">2</a></li>
  </ul>
    <button>Last</button><!-- can be disabled (not removed) if this is the last -->
    <button>Next</button><!-- can be disabled (not removed) if this is the last -->
</nav>


## Additional information

- Backdrop CMS version: 1.x

**Comment by jenlampton** [comment]
oops, this is a duplicate of https://github.com/backdrop/backdrop-issues/issues/5279. Closing

(not sure how I missed that the first time...)
